tests/upgradePMM_test.js

```diff
@@ -480,7 +480,7 @@ if (versionMinor >= 32) {
       I, settingsAPI, locationsAPI, backupAPI, scheduledAPI, inventoryAPI, backupInventoryPage, scheduledPage,
     }) => {
       if (!await inventoryAPI.apiGetNodeInfoByServiceName('MONGODB_SERVICE', mongoServiceName)) {
-        await I.say(await I.verifyCommand(`pmm-admin add mongodb --port=27027 --service-name=${mongoServiceName} --replication-set=rs0`));
+        await I.say(await I.verifyCommand(`pmm-admin add mongodb --port=27027 --service-name=${mongoServiceName} --replication-set=rs0 --cluster=rs0`));
```

Comment:
how dos this fix actually help

Reply:
this was the actual error - new backup services have to be a member of cluster. Looks like I've missed this test as it's pretty new